sysutils/starship/Portfile

```diff
@@ -15,9 +15,385 @@ description         a minimal, blazing fast, and extremely customizable prompt f
 long_description    Starship is ${description}.
 
 checksums           ${distname}${extract.suffix} \
-                    rmd160  24067510658627a84b9c197a0f95b97fc2267614 \
-                    sha256  142c88e462f590f0c66add67562e276085d2f0b7d448a3604e8f989c51653d7d \
-                    size    4291659
+                    rmd160  c27aead272c9ebc964bda36753958521fdd4ce57 \
+                    sha256  1748d61563df1051dbdc9e815de910bc30b34d0400056875b79a79420ce142fd \
+                    size    4327562 \
+                    aho-corasick-0.7.6.crate \
+                    rmd160  78c8c7e93bbcf52f761f90b7096cc9331ad3bfc6 \
+                    size    108953 \
+                    ansi_term-0.11.0.crate \
```

Comment:
Checksums for crates are added by `cargo.crates`, please don't set it again in `checksums`.

Reply:
The `cargo.crates` checksums only cover SHA256.

When `port lint` ran, it complained about not seeing the `rmd160` and `size` for every crate, so I added them.  If we don't care about `lint` then fine.

Reply:
You're right, but in this case please remove them. It would make the Portfile more complicated and harder to maintain.

Reply:
Done.